src/routes/PatronPreRegistrationRecordsContainer.js

```diff
@@ -43,6 +45,18 @@ const PatronPreRegistrationRecordsContainer = ({
     history.push('/users?sort=name');
   };
 
+  if (source) {
```

Comment:
Could calling `source.update` on every render cause any issues?

Reply:
Looks like `source` is also defined during render, so it should be ok. Withdrawing my comment